src/general/OptionsMenu.cs

```diff
@@ -2033,6 +2033,7 @@ private void OnMaxSpawnedEntitiesSelected(int index)
     private void OnMouseAxesBoundToggled(bool pressed)
     {
         mouseVerticalSensitivity.Editable = !pressed;
+        mouseVerticalSensitivity.FocusMode = pressed ? FocusModeEnum.Click : FocusModeEnum.All;
```

Comment:
While this probably works, could you also apply this state in the initial settings apply? Similarly to how I think I've remembered to do things for all of the settings. That's because when we eventually manage to fix the options menu causing extra control change callbacks on opening, this approach will stop working correctly.

Reply:
Sure